src/vcpkg/postbuildlint.cpp

```diff
@@ -750,17 +751,67 @@ namespace vcpkg::PostBuildLint
 
```

Comment:
We could save the "is header only" state here and use that to move the file to `share/pkgconfig`

Reply:
Because this is currently breaking folks in the main vcpkg repo, I don't want to change the "character" of the check substantially more than done here, Would you be willing to investigate in a followup PR?

Reply:
Yes I can do that